.github/actions/trivy/action.yml

```diff
@@ -39,28 +39,34 @@ runs:
       id: pull
       shell: bash
       run: |
+        mkdir -p $GITHUB_WORKSPACE/.cache/trivy/db
         oras pull ghcr.io/temporalio/trivy-db:2
+        tar -xzf db.tar.gz -C $GITHUB_WORKSPACE/.cache/trivy/db
+        rm db.tar.gz
+
+        mkdir -p $GITHUB_WORKSPACE/.cache/trivy/java-db
         oras pull ghcr.io/temporalio/trivy-java-db:1
+        tar -xzf javadb.tar.gz -C $GITHUB_WORKSPACE/.cache/trivy/java-db
+        rm javadb.tar.gz
```

Comment:
Is there a way to use their actions and not depend on their implementation details for where the DB locations should be found?